src/scripts.js

```diff
@@ -60,27 +68,48 @@ function createCards() {
       shortRecipeName = recipeInfo.name.substring(0, 40) + "...";
     }
     addToDom(recipeInfo, shortRecipeName)
+    //change that shit^
   });
+ 
+}
+
+function getTags(tags) {
+  let result = ''
+  tags.forEach(tag => {
+   let newTag = `<h4>${tag}</h4>` 
+   result += newTag
+  })
+return result
```

Comment:
2 spaces of indentation here please! :)

Our old style guide for mod 1 wants a space between any multi-line block and another statement, but since the lint doesn't care, neither do I.